rfp-proposals/ars-js-filecoin-explorer.md

```diff
@@ -0,0 +1,350 @@
+# RFP Proposal: `ars-js-filecoin-explorer`
+
+**Name of Project:** js-filecoin-explorer
+
+**Link to RFP:** [rfp-block-explorer](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-block-explorer.md)
+
+**RFP Category:** `app-dev`
+
+**Proposer:** [@waynewyang](https://github.com/waynewyang)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** Yes
+
+
+
+# Project Description
+
+The main purpose of js-filecoin-explorer is to give clearly views and general search functions for users of filecoin network, including clients, developers, miners, and investors. They can deeply know about filecoin network through js-filecoin-explorer.
+
+In order to meet needs of different users, we divide the filecoin blockchain explorer into four functional dimensions, 
+
+- Blockchain
+  - Including  Tipset, Block, Message, Actor.
+- Token
+  - Transaction history
+  - Token History
+    - Plan block rewards
+    - Real block rewards
```

Comment:
Why would planned and real block rewards be different? Can you say more about what this means?

Reply:
There was theoretical release curve for FIL, It perherps can't match with actual release counts every day.